backend/docker/Dockerfile.scheduler

```diff
@@ -2,14 +2,15 @@ FROM registry.access.redhat.com/ubi8/python-38
 
 # add application sources with correct perms for OCP
 USER 0
-COPY . /app
-WORKDIR /app
-RUN chown -R 1001:0 ./
+RUN dnf install -y libpq-devel python38-devel gcc
+ADD . /app
+RUN /usr/bin/fix-permissions /app
 USER 1001
 
 # Install deps
-RUN pip install --no-cache-dir -r requirements.txt
-RUN chgrp -R 0 ibutsu_server && chmod -R g+rwX ibutsu_server
+WORKDIR /app
+RUN pip install --no-cache-dir --upgrade pip wheel && pip install --no-cache-dir -r requirements.txt
+# RUN chgrp -R 0 ibutsu_server && chmod -R g+rwX ibutsu_server
```

Comment:
This is the main change standing out to me -- did you mean to leave this commented out?

Reply:
Yes, I actually meant to remove it. Higher in the file there's a command to fix the permissions.